lib/membrane_live_web/channels/event_channel.ex

```diff
@@ -197,16 +169,34 @@ defmodule MembraneLiveWeb.EventChannel do
     {:noreply, socket}
   end
 
+  @impl true
+  def handle_in("isPlaylistPlayable", _data, socket) do
+    case EventService.playlist_playable?(socket.assigns.event_id) do
+      {:error, _reason} ->
+        {:noreply, socket}
+
+      response ->
+        {:reply, {:ok, response}, socket}
+    end
+  end
+
   def handle_in(
-        "am_i_main_presenter",
-        %{"is_presenter" => is_presenter},
-        %{topic: "private:" <> topic_id} = socket
+        "last_viewer_answer",
```

Comment:
Why do we receive from js code strings in `snake_case`?

Reply:
I don't know why it's done like that. All messages from frontend messages are in snake case so I don't think I should refactor that in this particular PR